packages/wrangler/src/publish/publish.ts

```diff
@@ -496,6 +497,9 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			},
 			data_blobs: config.data_blobs,
 			durable_objects: config.durable_objects,
+			queues: config.queues.producers?.map((x) => {
+				return { binding: x.binding, queue_name: x.queue };
+			}),
```

Comment:
explicit naming of variables preferred when possible:
```suggestion
			queues: config.queues.producers?.map((producer) => {
				return { binding: producer.binding, queue_name: producer.queue };
			}),
```

Reply:
✅